docs/metrics-events.md

```diff
@@ -191,7 +198,8 @@ View name|Description
 `signin-unblock`|The sign-in unblock screen
 `choose-what-to-sync`|Choose what to Sync
 `connect-another-device`|Connect another device, phase 1
-`sms`|Connect another device, phase 2
+`sms`|Connect another device, phase 2, Send an SMS form
+`sms.sent`|Connect another device, phase 2, SMS sent
```

Comment:
While you're in here, are you able to furnish any of the empty rows above with descriptions? I have difficulty knowing which view is which in many cases and wasn't able to put a meaningful description to all of them